types/svg2png/index.d.ts

```diff
@@ -1,10 +1,18 @@
 // Type definitions for svg2png node package 
```

Comment:
Add the version: `// Type definitions for svg2png 4.1` (assuming that's the version this is for)